### ISSUE TITLE
Should we add a maximum-line-length rule of 80 to remarkrc?

### ISSUE THREAD

**Issue by jrhorn424** [ga-wdi-boston/meta#47]
In Atom, warnings are displayed for lines exceeding 80 characters. Using `remark-cli`, no warnings are displayed.

**Comment by raq929** [comment]
Yes. It's good practice to keep line length short, and it will make their
code readable if employers look at it.

**Comment by jrhorn424** [comment]
Of course, the problem is that remark is behaving inconsistently and I'm not sure who's wrong: me, remark, or Atom?

(I believe you may have missed this if responding directly from e-mail. I edited my OP, as I do quite frequently. Sorry!)

**Comment by raq929** [comment]
Aaah. I still think yes. Nothing wrong with explicitly stating the linting
rules. Does putting it in .remarkrc fix the problem you're having?
(I'm also having problems with my linter and Antony's not agreeing...)
